proto/zitadel/user/v2beta/query.proto

```diff
@@ -200,6 +201,16 @@ message InUserEmailsQuery {
     ];
 }
 
+//ResourceOwnerQuery always equals
+message ResourceOwnerQuery {
+    string resource_owner = 1 [
+        (validate.rules).string = {max_len: 200},
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            example: "\"69629023906488334\""
+        }
+    ];
+}
```

Comment:
please create a complete description (...always equals ?) and shouldn't the field be required / non-empty?
https://docs-git-user-v2-list-zitadel.vercel.app/docs/apis/resources/user_service_v3/user-service-list-users
![image](https://github.com/zitadel/zitadel/assets/9405495/55a9d856-573c-46ef-bc14-5bbd81e77254)